doc/reference/guidelines.rst

```diff
@@ -255,13 +303,13 @@ Inside these 3 groups, the imported lines are alphabetically sorted.
 Idioms
 -------
 
-- Prefer ``%`` over ``.format()``, prefer ``%(varname)`` instead of position (This is better for translation)
-- Try to avoid generators and decorators
-- Always favor *Readability* over *conciseness* or using the language features or idioms.
+- Prefer ``%`` over ``.format()`` (when only one variable to replace in a string), prefer ``%(varname)`` instead of position (when multiple variables are to be replaced in a string). This makes the translation easier for the translators community.
+- Avoid to create generators and decorators: only use the ones provide by the Odoo API.
+- Always favor *readability* over *conciseness* or using the language features or idioms.
```

Comment:
I see the point: there are cases where very concise implementations can turn out very hard to read.